### DISCUSSION TITLE
Laser will not turn on! It moves and everything but no laser. I checked all my connections and everything seems good, but still no laser!

### DISCUSSION THREAD

**Discussion by jokercardjunkie** [arkypita/LaserGRBL#1077] [Q&A]

**Comment by StuartB4** [comment]
What machine and laser is it, is it a new build or has it worked in the past. 
What version of GRBL and LaserGRBL do you have?

**Comment by jokercardjunkie** [comment]
Brand new machine. Ortur Laser GBL 2. Class 4 laser. I got it to run like it is supposed to operate but the laser doesn't come on. Very novice and confused!

**Comment by StuartB4** [comment]
As it's a new machine the best place might be the official Ortur Facebook group page. They seem to be really helpful. 
https://www.facebook.com/groups/orturusergroup
